src/api/research-study/gateways/EsResearchStudyRepository.test.ts

```diff
@@ -10,12 +8,13 @@ import { elasticsearchIndexMapping } from 'src/shared/elasticsearch/elasticsearc
 import { SearchResponse } from 'src/shared/elasticsearch/ElasticsearchService'
 import { RiphDtoTestFactory } from 'src/shared/test/helpers/RiphDtoTestFactory'
 
-describe('elasticsearch research study repository', () => {
+describe('elasticsearch research study repository', async () => {
+  const { elasticsearchService, esResearchStudyRepository, numberOfResourcesByPage } = await setup()
```

Comment:
**issue**: c'est problématique d'initialiser des globales en dehors d'un `it` et a fortiori après un `describe`. Ces globales existent partout dès lors que l'environnement de test est initialisé et que l'ensemble des noms de `describe` est lu.

Il est préférable de les scoper dans un `before` ou un `beforeEach`, ou bien conserver l'initialisation dans les `givens`